UnitTestForSevenRed/UnitTestsForSevenRed.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SevenRedLibrary;
+using System;
+using System.Collections.Generic;
+
+namespace UnitTestForSevenRed
```

Comment:
Add unit tests for `Card` class, to validate cards parsing
Ann unit tests to comparator method and for all ther logical parts of code